.github/workflows/workflow.yml

```diff
@@ -34,14 +32,18 @@ jobs:
           const branchName = context.payload.base_ref.replace('refs/heads/', '');
 
           const list = await github.rest.git.listMatchingRefs({
-            owner: 'volbobvol',
-            repo: 'action-target',
+            owner: ${{ env.TARGET_REPO_OWNER }},
+            repo: ${{ env.SUBMIT_SIGNING_REQUEST_TARGET_REPO }},
             ref: 'heads/' + branchName
           });
-          console.log(JSON.stringify(list));
+          
+          if (list.data.length === 0) {
+            console.log('Branch not found in the terget repository');
+          }
```

Comment:
Why is it necessary that the branch exists on the target repository?

Reply:
if a branch doesn't exist it means that some build has failed.
theoretically, all the branches should be synchronized.  
I can create a branch if it doesn't exist, but it will be fixing an error reason for which we don't know